src/dxvk/dxvk_adapter.cpp

```diff
@@ -274,6 +274,8 @@ namespace dxvk {
     VkPhysicalDeviceMemoryProperties memoryInfo = this->memoryProperties();
     
     Logger::info(str::format(deviceInfo.deviceName, ":"));
+    Logger::info(str::format("  Vendor ID: ", deviceInfo.vendorID));
+    Logger::info(str::format("  Device ID: ", deviceInfo.deviceID));
```

Comment:
These should be reported as hex numbers (`std::hex` is your friend).

Reply:
Done.